lib/puppet/face/facts.rb

```diff
@@ -1,5 +1,23 @@
 require 'puppet/indirector/face'
 require 'puppet/node/facts'
+require 'puppet/util/fact_dif'
+
+EXCLUDE_LIST = %w[facterversion swapfree_mb swapsize_mb load_averages\.15m
+  load_averages\.1m load_averages\.5m memory\.swap\.available_bytes
+  memory\.swap\.capacity memory\.swap\.total_bytes memory\.swap\.used_bytes
+  memory\.swap\.available memory\.system\.available
+  memory\.system\.available_bytes memory\.system\.capacity memory\.swap\.used
+  memory\.system\.total_bytes memory\.system\.used memory\.system\.used_bytes
+  memoryfree memoryfree_mb memorysize_mb mountpoints\..* mtu_.*
+  networking\.interfaces\..*\.mtu networking\.mtu partitions\..*\.filesystem
+  partitions\..*\.size_bytes partitions\..*\.mount partitions\..*\.uuid
+  system_uptime\.days system_uptime\.hours system_uptime\.seconds uptime_days
+  uptime_hours uptime_seconds disks\..*\.size_bytes
+  hypervisors\.lpar\.partition_number mountpoints\..*\.capacity
+  processors\.speed hypervisors\.xen\.privileged sp_uptime
+  system_profiler\.uptime hypervisors\.zone\..* system_uptime\.uptime uptime
+  hypervisors\.ldom\..* ldom_.* boardassettag dmi\.board\.asset_tag
+  blockdevice_.*_vendor blockdevice_.*_size is_virtual]
```

Comment:
Can we include `is_virtual`?

Reply:
Removed from list, as it is not volatile